python/lib/dependabot/python/update_checker/pip_compile_version_resolver.rb

```diff
@@ -216,10 +219,15 @@ def run_command(command, env: python_env)
           )
         end
 
+        def backwards_compat?
```

Comment:
Total nit, would this name make things clearer?

```suggestion
        def new_resolver_supported?
```

Reply:
Probably :)